src/ts/components/subtitlesettings/settinglist.ts

```diff
@@ -0,0 +1,30 @@
+import {Label} from '../label';
+import {FontColorSelectBox} from './fontcolorselectbox';
+import {FontOpacitySelectBox} from './fontopacityselectbox';
+import {FontFamilySelectBox} from './fontfamilyselectbox';
+import {FontSizeSelectBox} from './fontsizeselectbox';
+import {BackgroundColorSelectBox} from './backgroundcolorselectbox';
+import {BackgroundOpacitySelectBox} from './backgroundopacityselectbox';
+import {WindowColorSelectBox} from './windowcolorselectbox';
+import {WindowOpacitySelectBox} from './windowopacityselectbox';
+import {CharacterEdgeSelectBox} from './characteredgeselectbox';
+import {SettingsPanelItem} from '../settingspanel';
+import {SubtitleOverlay} from '../subtitleoverlay';
+import {Component, ComponentConfig} from '../component';
+
+// Helper function since the subtitle option panel should not change
+let GetSubtitleSettingList = (overlay: SubtitleOverlay): Component<ComponentConfig>[] => {
```

Comment:
Remove the `GetSubtitleSettingList` function and file (that is a bad pattern anyway) and add the subtitle settings items directly within the `SubtitleSettingsPanel` inside the constructor via a `mergeConfig` call (analogous to other components). The list is static and does not need to be changed from the outside.